( Templates )/Creatures/Shards Red&Gold/world/contentdb/templates/regular/actors/evil/d/shard_add.gas

```diff
@@ -0,0 +1,133 @@
+[t:template,n:shard_gold]

```

Comment:
I like to do this:
```
[t:template,n:foobar] { specializes = foobar_base; }
[t:template,n:foobar_base]
{...}
```
This way you can use the shard directly, and I can derive from it without making yours a non-leaf template, both still using the same implementation.